Userland/Libraries/LibC/sys/arch/i386/regs.h

```diff
@@ -25,8 +25,10 @@ struct [[gnu::packed]] PtraceRegisters : public __mcontext {
         return eip;
 #        elif ARCH(X86_64)
         return rip;
+#        elif ARCH(AARCH64)
```

Comment:
It's a bit weird that the x86_64 and aarch64 implementations are in the `arch/i368/regs.h` file. There is also a `arch/x86_64/regs.h` file, however feel free to ignore this comment.

Reply:
I agree, that's probably the wrong place for this, but I'm gonna leave it like that for now. :)